packages/packagers/html/src/HTMLPackager.js

```diff
@@ -47,13 +47,56 @@ export default new Packager({
     }, []);
 
     let {html} = await posthtml([
-      insertBundleReferences.bind(this, bundles)
+      insertBundleReferences.bind(this, bundles),
+      replaceInlineAssetContent.bind(
+        this,
+        getAssetContent.bind(this, bundleGraph, () => {})
+      )
     ]).process(code);
 
     return {contents: html};
   }
 });
 
+async function getAssetContent(bundleGraph, getBundleResult, assetId) {
+  let inlineBundle: ?Bundle;
+  bundleGraph.traverseBundles((bundle, context, {stop}) => {
+    if (bundle.id.includes(assetId)) {
```

Comment:
I would appreciate any recommendations on a better way to determine find the bundle based on the `assetId`.

Reply:
I’d look for bundle.getMainEntry().id.

Reply:
We could probably optimize this a bit too. Rather than traversing the whole graph to find each asset, you could use bundleGraph. findBundlesWithAsset and filter the result which should be quite a bit faster.

Reply:
Nice, I saw that and it looked like it needed an asset. I make an attempt to go that direction.

Reply:
I tried going to a `findBundlesWithAsset` approach, but the public `BundleGraph` doesn't have that function. 

`bundle.getMainEntry` should work though. However, it appears something recently changed on the v2 branch as I am getting some errors trying to find the inline bundles in the bundle graph.